compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/SwingPanel.desktop.kt

```diff
@@ -84,20 +83,16 @@ public fun <T : Component> SwingPanel(
         )
     }
 
-    EmptyLayout(focusSwitcher.backwardTrackerModifier)
-
     InteropView(
         factory = {
             interopViewHolder
         },
-        modifier,
+        modifier.then(focusSwitcher.modifier),
         update = {
             it.background = background.toAwtColor()
             update(it)
         }
     )
-
-    EmptyLayout(focusSwitcher.forwardTrackerModifier)
```

Comment:
Glad to see that it's finally gone. As far as I remember it blocked some unification between platforms before. Please add some TODO for me - I'll check that refactor later